hack/install-ocs-operator.sh

```diff
@@ -33,28 +33,39 @@ data:
 EOF
 
 
-# Ensure position independent make targets in release CI, explicitly setting the values ensures client-op doesn't deploy CSI
-# when storagecluster is configured for remoteconsumers, controllers set this value to "true"
-cat <<EOF | oc create -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: ocs-client-operator-config
-  namespace: openshift-storage
-data:
-  DEPLOY_CSI: "false"
-EOF
+patch_ocs_client_operator_config_configmap() {
+    while true; do
+        if oc get cm ocs-client-operator-config -n openshift-storage; then
+            oc patch cm ocs-client-operator-config -n openshift-storage --type merge -p '{"data":{"DEPLOY_CSI":"false"}}'
+            sleep 10
+            value=$(oc get cm ocs-client-operator-config -n openshift-storage -o custom-columns=:data.DEPLOY_CSI --no-headers)
```

Comment:
any reason in not using json? `oc get cm ocs-client-operator-config -nopenshift-storage -ojsonpath={.data.DEPLOY_CSI}`?

Reply:
Both do the same work.

Reply:
I understand the outcome will be same , however a li'l surprised w/ the choice and interested to know the reasoning.

Basically the current directive is creating single unnamed column w/ single row and again explicitly cutting the column name rather than directly picking the value of interest.

But fine if this is just a matter of preference.